packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -1,21 +1,27 @@
-import { IDesignSystem } from "../design-system";
+import designSystemDefaults, { IDesignSystem } from "../design-system";
 import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
 import { applyLocalizedProperty, toPx } from "@microsoft/fast-jss-utilities";
 import { typeRamp } from "../utilities/typography";
 import { IToggleClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { isEmpty } from "lodash-es";
 import * as Chroma from "chroma-js";
 import { Direction } from "@microsoft/fast-application-utilities";
 
 function setFloatAndClear(clearFloat: boolean = true): ICSSRules<IDesignSystem> {
     return {
-        float: (config: IDesignSystem): string => config.direction === Direction.ltr ? "left" : "right",
-        clear: clearFloat ? ((config: IDesignSystem): string => config.direction === Direction.ltr ? "left" : "right") : null
+        float: (config: IDesignSystem): string => {
+            return (isEmpty(config) || config.direction === Direction.ltr) ? "left" : "right";

```

Comment:
As before, I don't think isEmpty is safe, it may have other values but still be missing config.direction. Here I think you can just check for config.

Reply:
Let's land the above note on line 10 of text-field re: isEmpty() prior to digging in on this.